tests/storage/snapshot.go

```diff
@@ -17,6 +17,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/rand"
+	"k8s.io/client-go/util/retry"
```

Comment:
nice library, I think we might remove RetryIfModified from utils ;)

Reply:
I did not even know it existed (our own retry one).
Will you send a PR to remove it?

Reply:
I start vacation soon, I will not login for two weeks, so I do not want to start low priority PRs

Reply:
No problem, I will place it in my TODO.